src/containers/ConversionForm/index.tsx

```diff
@@ -224,7 +219,9 @@ const ConversionForm = ({
                         </button>
                     )}
                 </div>
-                <Divider orientation="right" orientationMargin={400}> </Divider>
+                <Divider orientation="right" orientationMargin={400}>
+                    {" "}
+                </Divider>
```

Comment:
Nit: 
```suggestion
                <Divider orientation="right" orientationMargin={400}/>
```

Reply:
Not sure if this requires child props but seems like it would be a little cleaner